source/extensions/extensions_build_config.bzl

```diff
@@ -14,11 +14,6 @@ EXTENSIONS = {
 
     "envoy.grpc_credentials.file_based_metadata":      "//source/extensions/grpc_credentials/file_based_metadata:config",
 
-    #
-    # WASM
-    #
-    "envoy.wasm":                                      "//source/extensions/wasm:config",
```

Comment:
@jplevyak it seems that this isn't necessary, since we're including each extension separately.

Reply:
extensions/wasm is independent of extensions/filter/http/wasm but they both depend on extensions/common/wasm.   And you can include a general service which just handles onsStart/onTIck by just using extensions/wasm.

I don't understand why you are moving extensions/wasm/config -> extensions/wasm/common/config
as  the former is for creating a general WASM service while the latter is the common library used by all
WASM things, i.e. AccessLog Filter::Http as well as the WASM service created in server.cc.

Let's chat tomorrow.

Reply:
I think the VmConfig should be on config/common because it is the shared bit among all the services and filters and the WasmConfig bit from config/wasm.proto should stay where it is as it is specificallyi the service config.